src/vmm/src/version_map.rs

```diff
@@ -63,6 +63,12 @@ lazy_static! {
 
     /// Static instance used for creating a 1:1 mapping between Firecracker release version
     /// and snapshot data format version.
+    /// !CAVEAT!
+    /// This map is supposed to be strictly one-to-one (i.e. bijective).
```

Comment:
Maybe we could briefly explain here why it has to be bijective? (e.g. describe-snapshot tries to invert it)

Reply:
Will do